tools/xmlcreate.py

```diff
@@ -126,13 +127,15 @@ def create_xml(src, name):
     with open(src,'r',encoding='utf-8') as srcfile:
         srcdata = srcfile.readlines()
     object = find_object(src)
-    xml = '<Root>\n    <File Name="' + object + '" Segment="6">\n'
+    xml = "<Root>\n    <File Name=" + "\"" + object +"\"" + " Segment=" + " \""+args.segment_number +"\""+">\n"
```

Comment:
Why the change from `'` to `"`? Seems to add unnecessary complexity to this line.